### PR TITLE
Oui field

### PR DESCRIPTION
**Pull request by neolitec**
Signed-off-by: Kevin Manson kevin.manson@corp.ovh.com

### CODE REVIEW

File: packages/oui-field/src/field.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<div class="oui-field">
+    <label class="oui-field__label oui-label"
+        ng-attr-for="{{ ::$ctrl.for }}"
+        ng-if="$ctrl.label"
```
Comment (tremblaymath) [inline packages/oui-field/src/field.html]: one time binding could be appropriate

File: packages/oui-field/src/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,355 @@
+describe("ouiField", () => {
```
Comment (tremblaymath) [inline packages/oui-field/src/index.spec.js]: Nice tests, easy to read !

I think that testing displayed messages could be interesting :
- to validate that error message descriptions settled in provider are used and well formatted with parameters.
- to validate that only one message is displayed with correct priority

File: packages/oui-field/src/field.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            <span ng-message="maxlength"
+                ng-bind="$ctrl.getErrorMessage('maxlength')"></span>
+            <span ng-message="min"
+                ng-bind="$ctrl.getErrorMessage('min')"></span>
+            <span ng-message="max"
+                ng-bind="$ctrl.getErrorMessage('max')"></span>
+        </span>
+    </div>
+    <div class="oui-field__helper"
+        ng-if="$ctrl.helpText"
```
Comment (tremblaymath) [inline packages/oui-field/src/field.html]: one time binding could be appropriate

File: packages/oui-field/src/field.html (hunk 1 above)
Comment (neolitec) [reply packages/oui-field/src/field.html]: One time binding has no effet with ng-if

File: packages/oui-field/src/field.html (hunk 3 above)
Comment (neolitec) [reply packages/oui-field/src/field.html]: +1

File: packages/oui-field/src/index.spec.js (hunk 2 above)
Comment (neolitec) [reply packages/oui-field/src/index.spec.js]: Yes, + parameters

**Pull request merged**